package.json

```diff
@@ -58,14 +58,14 @@
     "ember-maybe-import-regenerator": "^1.0.0",
     "ember-truth-helpers": "^2.0.0",
     "mdn-polyfills": "^5.16.0",
-    "xml-formatter": "^2.4.0"
+    "xml-formatter": "^2.4.0",
```

Comment:
@nvdk not sure how to deal with these
Ideally they'd be optional peer deps, but that's only a thing from node v7 and up
we could make them peerdeps and just ignore the warnings
looking at the syntax we may be able to already mark them as optional for when we do eventually switch to a new node (so we dont forget)

Reply:
FYI went this route, dont see any immediate issues with it, to be monitored